src/ResourceManagement/ServerManagement/ServerManagement.Tests/OperationsTest.cs

```diff
@@ -0,0 +1,359 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Management.ServerManagement;
+using Microsoft.Azure.Management.ServerManagement.Models;
+using Microsoft.Rest;
+using Microsoft.Rest.Azure;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace ServerManagement.Tests
+{
+    public class OperationsTest : ServerManagementTestBase
+    {
+        public OperationsTest(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task MakeSureTestsWork()
```

Comment:
!!

Seriously, I'm not sure what '??' means. 

If you're referring to the MakeSureTestsWork() test, I like to have that in there as a simple means of smoke-testing that the test framework itself is working. (especially when working in different contexts). 

It makes it much faster to call a single test that you're not concerned about it doing anything specific.